Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -759,18 +759,15 @@ public abstract class ConsoleDriver {
 		/// does not support displaying this rune.</returns>
 		public virtual bool IsRuneSupported (Rune rune)
 		{
-			if (rune.Value > RuneExtensions.MaxUnicodeCodePoint) {
-				return false;
-			}
-			return true;
+			return Rune.IsValid (rune.Value);
```

Comment:
We should just nuke the IsRuneSupported API and have callers use Rune.IsValid(rune.Value) instead.

Reply:
@tig I agree nuke the `IsRuneSupported` but also there is no need to have this callers in `AddRune` methods because the parameter of type `Rune` is always valid if this method is executed. Where can happen throwing an exception is when an API call `Driver.AddRune(invalid rune)` but never inside of the `AddRune` method. Please say something after you read this. Thanks.